cegs_portal/search/templates/search/v1/partials/_features.html

```diff
@@ -1,9 +1,8 @@
     {% load custom_helpers %}
 
     <div class="container min-w-full">
-        <div class="flex max-w-full min-w-full justify-between items-center">
-            <div class="justify-self-start"></div>
-            <div class="flex justify-center items-center grow table-title">
+        <div class="table-title" style="display: flex; align-items: center; justify-content: space-between; position: relative;">
+            <div style="display: flex; align-items: center; justify-content: center; flex-grow: 1;">
```

Comment:
You should use the appropriate tailwind classes when you can, instead of putting CSS in the style tag.